crates/typst-library/src/visualize/gradient.rs

```diff
@@ -1143,9 +1143,10 @@ impl Repr for ConicGradient {
 /// What is the gradient relative to.
 #[derive(Cast, Debug, Clone, Copy, PartialEq, Eq, Hash)]
 pub enum RelativeTo {
-    /// The gradient is relative to itself (its own bounding box).
+    /// The gradient or tiling is relative to itself (its own bounding box).
```

Comment:
```suggestion
    /// Relative to itself (its own bounding box).
```

It would probably be more future-proof to completely remove mentions of the places where this is used.

Reply:
> It would probably be more future-proof to completely remove mentions of the places where this is used.

I agree. Updated in bc7e946fe77c7077b709b2faee42257c03afb19d.